step1__select_samples.py

```diff
@@ -6,6 +6,7 @@
 from gnomad.sample_qc.sex import adjusted_sex_ploidy_expr
 from gnomad_qc.v3.resources.raw import get_gnomad_v3_mt
 from gnomad_qc.v3.resources.meta import project_meta
+from gnomad_methods.gnomad.utils.annotations import annotate_adj
```

Comment:
Should this just be `from gnomad.utils.annotations import annotate_adj`?

Reply:
yes!